src/state/completion.js

```diff
@@ -41,9 +41,10 @@ const completion = {
 			}
 		},
 		itemPercent(state, getters) {
+            let itemsCounted = Object.values(ITEMS).filter(itemId => !itemId.nocomplete);
 			let itemsComplete = Object.keys(ITEMS).filter(itemId => getters.getItem(itemId)).length;

```

Comment:
You probably need to filter this too; otherwise those "uncounted" items will contribute to the total that you've seen, and
it'll be possible to end up with >100% item completion (unless that's your intention)

Reply:
I'm not sure there. We did this at 2 AM and I'm pretty sure we decided that it doesn't really matter because they'd have to be cheated in. i"ll add this though, thank you!

Reply:
We were aware you could go >100% but deemed it a non-issue since these items should not be obtainable anyways

Reply:
I'm good with it getting you over 100%